ipc/cli/src/commands/wallet/balances.rs

```diff
@@ -50,16 +49,19 @@ impl CommandLineHandler for WalletBalances {
 
                 let v: Vec<anyhow::Result<(TokenAmount, &EthKeyAddress)>> = join_all(r).await;
 
-                for r in v.into_iter().filter_map(|r| r.ok()) {
-                    let (balance, addr) = r;
-                    if addr.to_string() != "default-key" {
-                        println!("{} - Balance: {}", addr.to_string(), balance);
-                        no_balance.retain(|a| a != addr);
+                for r in v.into_iter() {
+                    match r {
+                        Ok(i) => {
+                            let (balance, addr) = i;
+                            if addr.to_string() != "default-key" {
+                                println!("{} - Balance: {}", addr.to_string(), balance);
```

Comment:
I know we talked about this, but `jsonline` would be a good output format IMO, as it would make it easy to do something with the result using `jq`. A human readable output format is a great option, but personally I think it's liberating to print out structured data an defer the responsibility of formatting to the consumer.

Reply:
I kind of agree, but let's tackle this as a joint effort after LabWeek and let's track the final approach we want to follow here: https://github.com/consensus-shipyard/ipc/issues/365. So far depending on who is reviewing the PR they suggest one approach or the other :see_no_evil:

For this one specifically, this approach seems more readable for humans than returning a JSON.

Reply:
I'm sure it is more readable, and I agree you can postpone it. I guess like `ls -lh` you could have a "human readable hint" but it could also end up being an overkill.

Reply:
^ I like this idea of having a flag for human-readable v.s. computer-readable. Will track it in the issue :pray: